src/main/java/com/exadel/discountwebapp/location/entity/Location.java

```diff
@@ -0,0 +1,51 @@
+package com.exadel.discountwebapp.location.entity;
+
+import com.exadel.discountwebapp.vendor.entity.Vendor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Entity
+@Data
+@NoArgsConstructor
+@EqualsAndHashCode
+@Table(name = "location")
+public class Location {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Long id;
+
+    @Column(name = "country", length = 50, unique = true, nullable = false)
```

Comment:
not a unique

Reply:
@dmitriychernov7 unique constraint removed